### DISCUSSION TITLE
need help with select character

### DISCUSSION THREAD

**Discussion by loftyman123** [Lailloken/Exile-UI#145] [Q&A] [answered]
First of all great tool!

i did test it and i selected my testing character and everything worked for me.

but i did a second character to test what gonna hapen when i will league start and everything is ok except the fact i only have acces to the first character i made into the ''select character'' list. the restart option not on the list as the first time i did test it.

so my question is how do i add my new character to that list?

**Comment by Lailloken** [comment, marked as answer]
Characters will only show in the list after reaching level 2 because that is the first time they appear in the client's log-file. So you can simply select the character after killing hillock and restarting the script. Today's update will automatically add a new level 2 character to the list, without having to restart.
